src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -464,6 +559,7 @@ impl Node {
             );
         }
 
+        frp.set_error_status.emit(None);
```

Comment:
why is it needed?

Reply:
If we don't have it we get a black error indicator from the colour animation.